kuma/core/whitenoise/KumaDjangoWhiteNoise.py

```diff
@@ -0,0 +1,15 @@
+from whitenoise.django import DjangoWhiteNoise
```

Comment:
Please rename the file to `kuma/core/whitenoise.py` and then add the following as the first line to that file so it doesn't end up with a circular import error due to the equally named local module `whitenoise.py` and the package called `whitenoise`:

```python
from __future__ import absolute_import
```

That makes the feature that is in Python 3.x work on Python 2.x that we run under. More info here: https://www.python.org/dev/peps/pep-0328/